programs/validator-history/src/state.rs

```diff
@@ -52,15 +52,18 @@ pub struct ValidatorHistoryEntry {
     pub client_type: u8,
     pub version: ClientVersion,
     pub ip: [u8; 4],
-    // Required to keep 8-byte alignment
+    // Required so that `rank` is aligned such that curr_offset % 4 == 0 (u32 field.alignment) as per https://doc.rust-lang.org/reference/type-layout.html#reprc-structs
+    // without it - `rank` would have offset 27, and the compiler would add an implicit padding byte after `is_superminority` and before `rank`
     pub padding0: u8,
     // 0 if not a superminority validator, 1 if superminority validator
     pub is_superminority: u8,
     // rank of validator by stake amount
     pub rank: u32,
     // Most recent updated slot for epoch credits and commission
     pub vote_account_last_update_slot: u64,
-    pub padding1: [u8; 88],
+    // MEV earned
+    pub mev_earned: u64,
```

Comment:
in the interest of using the remaining space here wisely, I think it'd be best to represent this as a [fixed point decimal](https://en.wikipedia.org/wiki/Fixed-point_arithmetic). if this is 4 bytes with 2 decimals of precision that gives us a max MEV earned of 42949672.95 SOL for 1 validator for 1 epoch which won't happen for.. a long time 😄

Reply:
for sure, lmk if what i pushed up is correct

the internal u32 representation / 100 should give the max mev earned in sol with 2 dec of precision